package.json

```diff
@@ -39,10 +39,9 @@
     "babelify": "6.3.0",
     "browserify": "11.0.1",
     "concurrently": "2.0.0",
-    "jest-cli": "0.5.0",
     "reactify": "1.1.1",
     "uglifyify": "3.0.1",
     "watchify": "3.4.0",
-    "shrinkwrap": "6.0.1"
+    "npm-shrinkwrap": "6.0.1"
```

Comment:
I see a package called "shrinkwrap" that has version numbers starting with "0.", so I guess you meant "npm-shrinkwrap" here? Not sure...